# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/indexedDB/ActorsChild.h
# Commit: 296b2617fceb
# Full Hash: 296b2617fceb82bd87efffda2aabe38af402f7af
# Author: Simon Giesecke <sgiesecke@mozilla.com>
# Date: 2020-05-06 16:28:18
# Regressor Bug: 1623278
# File Overlap Count: 1
# Description:
#   Bug 1623278 - Make Factory derive from SafeRefCounted. r=dom-workers-and-storage-reviewers,janv
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D69398
# ==============================================================================

diff -r 74d93e32c149 -r 296b2617fceb dom/indexedDB/ActorsChild.h
--- a/dom/indexedDB/ActorsChild.h	Wed May 06 11:35:20 2020 +0000
+++ b/dom/indexedDB/ActorsChild.h	Wed May 06 12:20:14 2020 +0000
@@ -151,9 +151,9 @@
   // reference here?
   IDBFactory* mFactory;
 
-  NS_DECL_OWNINGTHREAD
+ public:
+  NS_INLINE_DECL_REFCOUNTING(BackgroundFactoryChild, override)
 
- public:
   void AssertIsOnOwningThread() const {
     NS_ASSERT_OWNINGTHREAD(BackgroundFactoryChild);
   }